### PR TITLE
Revised CHC terms

### PR DESCRIPTION
**Pull request by bweaver-work**

### CODE REVIEW

File: docs/api/covidcast-signals/changehc.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ us by Change Healthcare. Change Healthcare is a healthcare technology company
 that provides us with information about outpatient visits, which they aggregate
 from many healthcare providers. Using this outpatient data, we estimate the
 percentage of COVID-related doctor's visits in a given location, on a given day.
+Change Healthcare claims data that has been de-identified in accordance with
+HIPAA privacy regulations.
```
Comment (capnrefsmmat) [inline docs/api/covidcast-signals/changehc.md:22]: This isn't a complete sentence. Did you mean to work it in somewhere else?

Comment (krivard) [reply docs/api/covidcast-signals/changehc.md:22]: sorry, this is just a placeholder containing the phrasing Change has requested we use

Comment (bweaver-work) [reply docs/api/covidcast-signals/changehc.md:22]: This was just me getting a PR started for Katie to update terms today, I'm not sure where it's going yet.

**Comment by krivard** [comment]
Superseded by #285 and #286.

**Pull request closed without merging**